gcc/rust/expand/rust-expand-visitor.cc

```diff
@@ -149,6 +149,63 @@ expand_attribute (std::unique_ptr<AST::Item> &item, AST::SimplePath &name,
   return result;
 }
 
+template <typename T>
+static std::vector<std::unique_ptr<AST::Stmt>>
+expand_stmt_attribute (T &item, AST::SimplePath &name, MacroExpander &expander)
```

Comment:
I feel like this function needs a comment with a small example its not clear what types of T will be valid here because C++ is not as good as Rust lol

Reply:
Fixed in [26fe2f9](https://github.com/Rust-GCC/gccrs/pull/2274/commits/26fe2f9120795cb8b53a53c39f82904570cde8d9)